postgresql/README.md

```diff
@@ -1,6 +1,6 @@
 # PostgreSQL
 
-Docker image for [PostgreSQL] version 15.1
+Docker image for [PostgreSQL] version 16.3
```

Comment:
I got 16.2 when I checked.

```
/ $ postgres --version
postgres (PostgreSQL) 16.2
```

Reply:
Hmmmm I'm not getting the same results from a fresh build?

* Reran `make clean && make bake && make up`

**Check 1:**

 ```bash
docker run --rm -d --name postgresql islandora/postgresql

➜  isle-buildkit git:(july-2024-update) docker exec -ti postgresql psql -U root postgres
psql (16.3)
Type "help" for help.

postgres=#
```

Check 2:
```bash
docker exec -ti postgresql bash

6f89c38c08af:/# postgres --version

postgres (PostgreSQL) 16.3
```

Reply:
My local build matches the version from the latest release:

```
make bake BUILDER=desktop-linux TARGET=postgresql 2&>/dev/null && docker run --rm -ti -u 1 --entrypoint psql islandora/postgresql:local --version
psql (PostgreSQL) 16.2
```

So, I obliterated all my local builds and then built without access to the cache on DockerHub and got what you have. Which is worrying, I'll open up a ticket to investigate this cache poisoning issue.